ddtrace/internal/_encoding.pyx

```diff
@@ -922,75 +901,6 @@ cdef class MsgpackEncoderV05(MsgpackEncoderBase):
 
         return 0
 
-    cpdef _verify_encoding(self, encoded_bytes):
```

Comment:
The only change I see here is that the encoded payload is not verified (which was where msgpack was being imported).

Reply:
This changes reverts verifying encoding. This feature is reliant on the msgpack dependency. By removing this feature we no longer need to import msgpack the import errors disappear.